### DISCUSSION TITLE
All videos on youtube work except this one

### DISCUSSION THREAD

**Discussion by NealStrassner** [aclap-dev/video-downloadhelper#1073] [Help and Q&A]
### Your question

https://www.youtube.com/watch?v=EuKkkZkVS2A  Why can the plugin not see this one but it can all others?

### Addon & CoApp version and details

Version 9.0.2.12 (stable), target mozilla, locale en-US.

Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3958, Firefox 128.0.3 64-bit, licensed VDH 9.1.0.39 beta, CoApp 2.0.19.

![#01](https://github.com/user-attachments/assets/e7cb96ce-8f51-48ad-a840-631dc93626e9)
![#02](https://github.com/user-attachments/assets/43b627f6-4c09-4c83-8b21-82272eb0d464)
![#03](https://github.com/user-attachments/assets/0f90541d-c3dd-4d04-93e7-7c299510f860)
![#04](https://github.com/user-attachments/assets/4674e929-28af-43fd-b313-6a467d81b753)
![#05](https://github.com/user-attachments/assets/d38409b7-06a9-4c16-a2fe-7b9e062ac2c1)

Maybe the beta would cure your problem.

**Comment by paulrouget** [comment]
We're getting more and more of these reports. I'm gonna start collecting them here: https://github.com/aclap-dev/video-downloadhelper/discussions/1074
